lib/pbench/test/unit/agent/task/test_tools_kill.py

```diff
@@ -176,18 +176,18 @@ def iterdir(self):
             [
                 ("abc.file", None),  # top level file
                 ("filenotf", None),  # directory with no .uuid file
-                ("run1", "abcdef"),  # directory with a file
-                ("run2", "ghijkl"),  # directory with a file
+                ("res1", "abcdef"),  # directory with a file
+                ("res2", "ghijkl"),  # directory with a file
             ]
         )
-        pairs = list(kill.gen_run_directories(pr))
+        pairs = list(kill.gen_result_tm_directories(pr))
```

Comment:
It would probably be good to confirm that the correct number of pairs is returned.